src/core/annotation.js

```diff
@@ -617,14 +623,63 @@ class MarkupAnnotation extends Annotation {
     super(parameters);
     const dict = parameters.dict;
 
-    if (!dict.has('C')) {
-      // Fall back to the default background color.
-      this.data.color = null;
+    this.data.isMarkup = true;
+
+    if (dict.has('IT')) {
+      this.data.intent = dict.get('IT').name;
+    }
+
+    if (dict.has('IRT')) {
+      this.data.inReplyTo = dict.getRaw('IRT').toString();
+      this.data.replyType =
+      dict.has('RT') ? dict.get('RT').name : AnnotationReplyType.REPLY;
     }
 
-    this.data.hasPopup = dict.has('Popup');
-    this.data.title = stringToPDFString(dict.get('T') || '');
-    this.data.contents = stringToPDFString(dict.get('Contents') || '');
+    if (this.data.replyType === AnnotationReplyType.GROUP) {
+      const parent = dict.get('IRT');
```

Comment:
Let's add a comment after the `if` statement: "Subordinate annotations in a group should inherit the group attributes from the primary annotation." (taken from the spec)